pkg/cmd/template/schema_test.go

```diff
@@ -684,8 +686,10 @@ schema.yml:4 |   subnet_ids: null
 }
 
 func TestSchemaFeatureIsNotEnabledButSchemaIsPresentReportsAWarning(t *testing.T) {
-	opts := cmdtpl.NewOptions()
```

Comment:
how are these tests passing if we deleted opts and it's being used on the next line?

Reply:
ah i see -- https://github.com/vmware-tanzu/carvel-ytt/pull/305/files#diff-2ab902ad23ffe96207ff6466403834df5538e509639024bfc503aed26a377cb8R22

having a global variable even in tests isnt great. how can we do this better?

Reply:
Agreed, the global is removed as part a refactor in #301.